dash/include/dash/Shared.h

```diff
@@ -83,7 +105,7 @@ class Shared {
       _owner,
       _team->dart_id());
     team.barrier();
```

Comment:
That barrier is superfluous.

Reply:
Thanks for the information.

I left it alone as it was there before I tempered with the file.
It is removed now.